docs/overview.md

```diff
@@ -102,10 +104,10 @@ These provide the most searchable and comprehensive documentation on LDK.
 If you're using Java and want more information on any method/struct/etc., searching
 the Rust docs for the Rust version of that struct/method is your best bet.
 
-### [Rust Sample Node](https://github.com/TheBlueMatt/rust-lightning-bitcoinrpc)
+### [Rust Sample Node](https://github.com/lightningdevkit/ldk-sample)
 
-While this node is a little outdated, it's still a very useful reference for how to construct
-a lightning node using LDK.
+The sample serves as a complete reference for constructing a lightning node with
+the LDK. This is a good starting point if you want a self-guided tour!
```

Comment:
In other parts of the docs I think we've been saying "LDK" instead of "the LDK." Doesn't matter too much though.

Doing 1 last pass before merge!